app/views/locations/index.html.erb

```diff
@@ -0,0 +1,110 @@
+<div class="row">
+  <div class="large-4 small-12 columns">
+
+    <img src="http://placehold.it/500x500&text=Logo">
+
+    <div class="hide-for-small panel">
+      <h3>Header</h3>
+      <h5 class="subheader">Risus ligula, aliquam nec fermentum vitae, sollicitudin eget urna. Donec dignissim nibh fermentum odio ornare sagittis.
+      </h5>
+    </div>
+
+    <a href="#">
+    <div class="panel callout radius">
+      <h6>99  items in your cart</h6>
+    </div>
+    </a>
```

Comment:
What's this stuff here? This looks suspiciously copied-and-pasted from a template... As far as I know you guys won't be selling anything! Be careful not to get too indiscriminate with copy-and-paste, and to only submit stuff that you actually want to have. Maybe get rid of this part!